core/src/main/java/nl/nn/adapterframework/unmanaged/SpringJmsConnector.java

```diff
@@ -279,7 +279,6 @@ public void onMessage(Message message, Session session)	throws JMSException {
 				getReceiver().processRawMessage(listener, message, pipeLineSession, false);
 				if (log.isTraceEnabled()) log.trace("transaction status after processRawMessage: {}", displayTransactionStatus(txStatus));
 			} catch (ListenerException e) {
-				getReceiver().increaseRetryIntervalAndWait(e, logPrefix);
```

Comment:
Dan kan de methode ook gelijk private gemaakt worden.
Even toch voor de zekerheid, dit was een dubbele `increaseRetryIntervalAndWait` ?

Reply:
Deze werd aangeroepen wanneer eerder de vlag aanstond waardoor `hasProblematicHistory()` werd overgeslagen, en daardoor was het geen dubbele aanroep van `increaseRetryIntervalAndWait()`.

Ik zal kijken of het beter is om deze method `private` te maken, of om de dubbele implementatie hiervan weg te doen die zit in de `PullingListenerContainer`? Of is dat teveel om nu nog in 7.8 proberen te krijgen?